plugins/lookup/hashi_vault.py

```diff
@@ -147,13 +147,15 @@
         - approle
         - aws_iam_login
         - jwt
+        - none
       default: token
     return_format:
       description:
         - Controls how multiple key/value pairs in a path are treated on return.
         - C(dict) returns a single dict containing the key/value pairs.
         - C(values) returns a list of all the values only. Use when you don't care about the keys.
         - C(raw) returns the actual API result (deserialized), which includes metadata and may have the data nested in other keys.
+        - C(none) auth method was added in collection version C(1.2.0).
```

Comment:
This needs to go to `auth_method`. Right now this is in the description of `return_format`.

Reply:
oof, good find, thank you!